extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/KeycloakPolicyEnforcerConfig.java

```diff
@@ -35,7 +35,7 @@
          * Enables policy enforcement.
          */
         @ConfigItem
-        boolean enable;
+        public boolean enable;
```

Comment:
In `OidcConfig` we have the 'getters' pattern so I'd propose to do it here as well as opposed to have some properties public and some not, hope it works for you

Reply:
public properties is ok, most extensions just use properties

Reply:
Sounds good then